lib/ddtrace/tracer.rb

```diff
@@ -215,6 +257,7 @@ def publish(trace)
     #
     #   tracer.shutdown!
```

Comment:
⬆️ This and a few more methods on this class have code examples that are outside ```` ``` ```` blocks and thus don't get monospaced/syntax highlighted.

Reply:
Fixed two cases of examples outside of a code block in this file.